modules/core/shared/src/main/scala/lucuma/core/model/SiderealTracking.scala

```diff
@@ -73,24 +73,24 @@ object SiderealTracking extends SiderealTrackingOptics {
     * Proper motion correction in model units.
     * @param baseCoordinates base coordinates
     * @param epoch           the epoch
-    * @param properVelocity  proper velocity per epoch year
+    * @param properMotion  proper velocity per epoch year
     * @param radialVelocity  radial velocity (km/sec, positive if receding)
     * @param parallax        parallax
     * @param elapsedYears    elapsed time in epoch years
     * @return Coordinates corrected for proper motion
     */
-  def properMotion(
+  def properMotionOn(
```

Comment:
I renamed this to avoid a name clash with generated lenses, but maybe a better name would be `coordinatesOn`?

Reply:
Right.  We're getting `Coordinates` back so maybe `correctedCoordinates` or `adjustedCoordinates`?